Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN pip install --no-cache-dir -r requirements.txt
 
 # Install Node requirements
 COPY ./package.json /app/package.json
-RUN npm install
+# RUN npm install
```

Comment:
For some reason docker build fails on this step and I couldn't figure out how to solve it. Commenting it out didn't break the app, so this is my Dockerfile solution for now.